runtimes-common/attributes/runtimes-attributes.adoc

```diff
@@ -0,0 +1,44 @@
+//
+// This file contains attributes for Red Hat Runtimes product docs.
+//
+
+//
+//Metering labels: product specific
+//
+
+//Each product team must uniquely define two attributes in their projects.
+//This file gets automatically overwritten so treat it as READ only!!
```

Comment:
@oraNod One clarification / question for you yet, if possible. Per the comment above this file is to be treated `read-only`, but there are three attributes required to be defined:

```
:ProductName: {project_name}
:component-name: {project_name}
:component-version: {project_version_base}
```

the documentation to be build / rendered correctly. Can we defined those three here (and remove their definition from `maste.adoc` in that case), or do we need to consider the `runtime-attributes.adoc` file as immutable (is there a chance it will get updated on `sync with upstream` on read-only) basis again?